src/wp-includes/feed.php

```diff
@@ -482,6 +482,10 @@ function rss_enclosure() {
 			foreach ( (array) $val as $enc ) {
 				$enclosure = explode( "\n", $enc );
 
+				if ( 3 > count( $enclosure ) ) {
```

Comment:
Yoda required for function calls or greater/less than comparisons as they are less clear.

```suggestion
				if ( count( $enclosure ) < 3 ) {
```